### ISSUE TITLE
Extra spacing on integrated terminal

### ISSUE THREAD

**Issue by cossio** [microsoft/vscode#159550]
Type: <b>Bug</b>

Open a new integrated terminal (Ctrl + `). There is a weird extra-spacing, see:

<img width="603" alt="Screenshot 2022-08-30 at 13 18 00" src="https://user-images.githubusercontent.com/10637482/187423667-0dd06981-0ffe-4755-8042-530ee5ea5b2a.png">

VS Code version: Code 1.70.2 (Universal) (e4503b30fc78200f846c62cf8091b76ff5547662, 2022-08-16T05:36:37.829Z)
OS version: Darwin arm64 21.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M2 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|24.00GB (0.26GB free)|
|Process Argv|--crash-reporter-id ed46a0f1-8942-4505-a0ac-7dd97281b81b|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
better-toml|bun|0.3.2
mdmath|goe|2.7.4
language-julia|jul|1.7.6
python|ms-|2022.12.1
vscode-pylance|ms-|2022.8.40
jupyter|ms-|2022.7.1102252217
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.9
remote-containers|ms-|0.245.2
remote-ssh|ms-|0.84.0
remote-ssh-edit|ms-|0.80.0
remote-wsl|ms-|0.66.3
vscode-remote-extensionpack|ms-|0.21.0
vsliveshare|ms-|1.0.5696
vsliveshare-audio|ms-|0.1.91
vscode-todo-highlight|way|1.0.5

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonvspyt551:30345470
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
pylanb8912:30545647
vsclangdc:30486549
c4g48928:30535728
hb751961:30553087
dsvsc012:30540252
azure-dev_surveyone:30548225
i497e931:30553904

```

</details>

**Comment by KathirvelChandrasekaran** [comment]
This might be related to the terminal's font. You can change the font in **Preferences: Open User Settings (JSON)**

`"terminal.integrated.fontFamily": "monospace"`

**Comment by bdougherty** [comment]
I've also had this issue. It was introduced a version or two back. I have not changed any of my fonts, and the font is valid ([this comment](https://github.com/microsoft/vscode/issues/155442#issuecomment-1196018019) on a previous similar bug is incorrect).

**Comment by Noxillio** [comment]
I experience this issue as well. Font change is not a permanent fix, nor is changing the spacing to 1 and back. Will try keeping it at 1 but I'd really prefer not to have to do that.

**Comment by bdougherty** [comment]
Seems to have been fixed for me in 1.72.0
